docs/app/Examples/elements/Input/Variations/Labeled.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import {Input} from 'stardust';
+
+export default class Labeled extends Component {
+  render() {
+    return (
+      <Input className='labeled' placeholder='mysite.com'>
+        <div className='ui label'>
```

Comment:
See Issue #61 - currently Input can't handle a label. This is causing it to render weird where the button can't go on the left side.